tests/disaster-recovery/regional-dr/test_failover_and_relocate.py

```diff
@@ -201,6 +241,44 @@ def test_failover_and_relocate(
         logger.info(f"Waiting for {wait_time} minutes to run IOs")
         sleep(wait_time * 60)
 
+        # Get lastGroupSyncTime after failover
+        for drpc_data in TimeoutSampler(60, 3, drpc_obj.get):
```

Comment:
The timeout could be further increased as each and every image has to sync before lastGroupSyncTime would have a value, and hence is likely to take more time.

Reply:
Increased the timeout.